plugins/jdg66/pom.xml

```diff
@@ -166,20 +166,8 @@
            <artifactId>radargun-mapreduce</artifactId>
            <version>${project.version}</version>
            <scope>provided</scope>
-       </dependency>     
-       
-   </dependencies>
+       </dependency>
 
-   <dependencyManagement>
-      <dependencies>
-         <dependency>
-            <groupId>org.infinispan</groupId>
-            <artifactId>infinispan-parent</artifactId>
```

Comment:
My intent with adding this dependency was to avoid needing the `property` settings above. This avoid us needing to maintain the versions manually.

Reply:
Exactly. The main issue is that those old poms or a plugin have a dependency for `${java.home}/../lib/tools.jar`. Instead of drop JDK8 support, I decided that it is better to allow to build with JDK11 and keep JDK8 support adding a property value.

Reply:
Well, the `tools.jar` file is only used by the `JavaProcessService` class, and I'm still not clear why removing this dependency helps?

Reply:
Because there are plugins from `infinispan-parent` that use it.
For example: in this PR you will see that the plugins are using the latest version because of the same issue.

Reply:
Sorry, @diegolovison I am still confused by this statement. Can you just explain what you are fixing by removing the `infinispan-parent` dependency from the plugin POM file? If a plugin expects to be running in a JDK during build time, that is fine. Setting `java.home` for the `JavaProcessService` is a runtime concern, so I don't see how those two are related?

Reply:
Plugins from `<artifactId>infinispan-parent</artifactId>` are using `tools.jar`.
With the `<artifactId>infinispan-parent</artifactId>` the build with JDK11 will fail.

Reply:
Well, I don't tell that `JavaProcessService` is related with `<artifactId>infinispan-parent</artifactId>` :)